spring-cloud-task-integration-tests/src/test/java/org/springframework/cloud/task/listener/BatchExecutionEventTests.java

```diff
@@ -240,7 +241,7 @@ public void receive(Object exceptionMessage) {
 	public static class ItemWriteEventsListenerBinding {
 
 		@StreamListener(Sink.INPUT)
-		public void receive(Object itemWrite) {
+		public void receive(String itemWrite) {
 			assertTrue("Message should start with '3 items'", itemWrite.toString().startsWith("3 items "));
```

Comment:
@mminella The test failures were called by a regression - see https://github.com/spring-cloud/spring-cloud-stream/issues/844. 

On the other hand, I would recommend keeping these changes, as they are closer to what a user would be expected to do anyway (i.e. not use Object and downcast, but use the actual expected type as argument).